src/app/main/main.component.js

```diff
@@ -44,6 +46,10 @@ function routingConfig($stateProvider, $locationProvider, $urlRouterProvider){
       url: "/thank-you.html",
       template: '<thank-you></thank-you>'
     })
+    .state('profile', {
+      url: "/profile.html",
```

Comment:
I guess the default profile page doesn't exist yet. You can use query params for now but idk what the final version should use. @adammeyer What links do we want to use in AEM? Do we have a profiles directory like `give.cru.org/profile/payment-methods.html` or is everything based off the root folder as its own html file?